### PR TITLE
Add new option want_trove

### PR DESCRIPTION
**Pull request by mjura**

### CODE REVIEW

File: scripts/qa_crowbarsetup.sh [hunk 1]
```diff
@@ -3334,6 +3335,7 @@ function deploy_single_proposal()
             ;;
         trove)
             iscloudver 5plus || return
+            [[ $want_trove = 1 ]] || return
```
Comment (nicolasbock) [inline scripts/qa_crowbarsetup.sh]: Shouldn't this be testing for 0?

Comment (mjura) [inline scripts/qa_crowbarsetup.sh]: This is correct, this is equal to
```
if [[ $want_trove != 1 ]]; then
  return
fi
```
We don't want to apply trove proposal when option want_trove is different than 1

**Comment by bmwiedemann** [comment]
needs rebase

File: scripts/qa_crowbarsetup.sh (hunk 1 above)
Comment (jdsn) [inline scripts/qa_crowbarsetup.sh]: I guess @nicolasbock was asking if it makes more sense to phrase it in a positive way like:
` [[ $want_trove = 0 ]] && return`

Same question may apply to the line above:
`iscloudver 4minus && return`

But thats a matter of taste.

Comment (nicolasbock) [inline scripts/qa_crowbarsetup.sh]: @jdsn Thanks for putting some more meat into my question :) I always get confused with the style used above until I remember that it's a `||` and not an `&&`.

**Comment by mjura** [comment]
It was rebased

**Comment by aplanas** [comment]
Uhmm, the gate fail was related with NFS fail. Rebuilding.

**Pull request merged**